elements/pl-drawing/info.json

```diff
@@ -2,12 +2,11 @@
     "controller": "pl-drawing.py",
     "dependencies": {
         "nodeModulesScripts": [
-           "fabric/dist/fabric.js"
+            "fabric/dist/fabric.js",
+            "lodash/lodash.js"
         ],
         "coreScripts": [
-            "sylvester.js",
-            "sha1.js",
-            "underscore.js"
+            "sylvester.js"
```

Comment:
Is `sylvester.js` actually used in `pl-drawing`? @nicknytko?

Reply:
It is, removing it breaks the question.

Reply:
https://github.com/PrairieLearn/PrairieLearn/blob/master/elements/pl-drawing/mechanicsObjects.js#L4

Reply:
Yes, its used for vector math

Reply:
Cool, thanks for confirming!